api/server/server.go

```diff
@@ -884,6 +884,11 @@ func (s *Server) bindHandlers(ctx context.Context) {
 			v1.GET("/apps", s.handleAppList)
 			v1.POST("/apps", s.handleAppCreate)
 
+			v1.GET("/funcs", s.handleFuncsList)
```

Comment:
Should we just chuck all this under /v0.2 or something similar?

Reply:
that was an option, i think we decided amongst ourselves that since v1 isn't really v1 we could just use v1

Reply:
thinking of terms of allowing both the new and old to sit side by side for a bit, but maybe big bang is the way to go